src/main/java/microsoft/exchange/webservices/data/MessageBody.java

```diff
@@ -100,7 +105,15 @@ protected void readAttributesFromXml(EwsServiceXmlReader reader)
   @Override
   protected void readTextValueFromXml(EwsServiceXmlReader reader)
       throws XMLStreamException, ServiceXmlDeserializationException {
-    this.text = reader.readValue();
+    if (log.isDebugEnabled()) {
+       log.debug("Reading text value from XML. BodyType = " + this.getBodyType() +
+          ", keepWhiteSpace = " +
+          ((this.getBodyType() == BodyType.Text) ? "true." : "false."));
```

Comment:
I intentionally used String concat here because
- it's about 10 times faster than String.format()
- it's used in other ews-java-api log messages already, see SimpleServiceRequestBase.java:123:
```
      log.error("Response content type not XML; first line: '" + line + "'");
```
So I will not change it.